rules/decoder-events.rules

```diff
@@ -143,6 +143,3 @@ alert pkthdr any any -> any any (msg:"SURICATA ERSPAN too many vlan layers"; dec
 # Cisco Fabric Path/DCE
 alert pkthdr any any -> any any (msg:"SURICATA DCE packet too small"; decode-event:dce.pkt_too_small; classtype:protocol-command-decode; sid:2200110; rev:2;)
 
-alert pkthdr any any -> any any (msg:"SURICATA packet with too many layers"; decode-event:too_many_layers; classtype:protocol-command-decode; sid:2200111; rev:1;)
```

Comment:
we still need this rule, just need it to have a unique sid. Also, we should probably make sure it has the same sid across our supported branches. CI checks for both would be great :)

Reply:
increased sid, looks like that's what it is in other branches.

will look into github CI for testing rules